src/scripts/index.js

```diff
@@ -1,56 +1,75 @@
-let books = [];
 const addButton = document.querySelector('.add-btn');
 const bookList = document.querySelector('.books-ul');
 const storage = window.localStorage;
 
-const Book = (name, title) => {
-  this.name = name;
-  this.title = title;
-};
+class BookHandler {
+  constructor() {
+    this.book = '';
+    this.books = [];
+    this.newBook = {
+      name: '',
+      title: '',
+    };
+  }
 
-const addBook = () => {
-  const bookName = document.querySelector('.bookname').value;
-  const bookTitle = document.querySelector('.author').value;
+  addBook = () => {
+    const bookName = document.querySelector('.bookname').value;
+    const bookTitle = document.querySelector('.author').value;
 
-  const book = new Book(bookName, bookTitle);
-  books.push(book);
-  storage.setItem('books', JSON.stringify(books));
-};
+    this.newBook = {
+      name: bookName,
+      title: bookTitle,
+    };
+    this.books.push(this.newBook);
+    storage.setItem('books', JSON.stringify(this.books));
+  };
 
-const showBooks = () => {
-  let book = '';
-  if (books.length === 0) {
-    bookList.innerHTML = '<p>Sorry you have no book left. Kindly add some</p>';
-  } else {
-    books.forEach((bookObj, ind) => {
-      book += `<li class="book-li">
+  showBooks = () => {
+    if (this.books.length === 0) {
+      bookList.innerHTML = '<p>Sorry you have no book left. Kindly add some</p>';
+    } else {
+      this.book = '';
+      this.books.forEach((bookObj, ind) => {
+        this.book += `<li class="book-li">
     <span>${bookObj.name}</span> <span>${bookObj.title}</span>
     <input type="button" class="btn rmv" data-target=${ind} value="Remove" />
   </li>`;
-      bookList.innerHTML = book;
-    });
-  }
+        bookList.innerHTML = this.book;
+      });
+    }
+  };
+
+  rmvBook = (e) => {
+    this.books = this.books.filter(
+      (book, index) => index !== Number(e.target.attributes[2].value),
+    );
+  };
+
+  loadBooks = () => {
```

Comment:
- [ ] [REQUIRED] I still got an error on my console when I open your APP on my browser for the first time. It will be nice if you work on it.
![1](https://user-images.githubusercontent.com/58552748/144370325-1dbb8139-274c-4daa-9bbf-eb2f6e9d23fa.jpg)

Reply:
okay, thanks for the review, but when you get an error from a code you are reviewing, I believe the proper thing to do is to describe the processes that lead to the error. To just say I got this error in my browser doesn't make sense. Which browser are you using? How did you arrive at the error and are there any tips to solve the error? I was not getting any errors because I was using Chrome. You can understand how that would be frustrating to someone with limited coding experience.
![firefox](https://user-images.githubusercontent.com/17967537/144566531-f46af6f3-dac7-4783-81
![chrome error](https://user-images.githubusercontent.com/17967537/144566595-3c03fa54-2a39-4954-a7ed-431f7dee6cd4.png)
![firefox](https://user-images.githubusercontent.com/17967537/144566607-3931ec83-4154-4f72-95f9-886f69bd0000.png)
88-fb1566bfee37.png)